Duplicati/Library/Main/Options.cs

```diff
@@ -1620,11 +1642,22 @@ public bool Dryrun
         }
 
         /// <summary>
-        /// Gets a flag indicating if the remote verification is deep
+        /// Gets a value indicating if the remote verification is deep
         /// </summary>
-        public bool FullRemoteVerification
+        public RemoteTestStrategy FullRemoteVerification
         {
-            get { return Library.Utility.Utility.ParseBoolOption(m_options, "full-remote-verification"); }
+            get
+            {
+                string policy;
+                if (!m_options.TryGetValue("full-remote-verification", out policy))

```

Comment:
To best preserve the old behavior, `--full-remote-verification` (with an empty string in the options dict) would need to result in true. Otherwise, existing configurations might silently change from true to false. 
However, this is different from the other enum options and might lead to other confusion. Maybe check for an empty string, set the result to true and print a warning that the option was changed and true should be specified explicitly (also I think an invalid enum should also print a warning).

Reply:
@Jojo-1000 
what do you mean by 'an empty string in the options dict' ?
There is no dict in the database, only rows. If the option is true in the Web UI, we get this in the Option table:
1||--full-remote-verification|true
if it false, we get this:
1||--full-remote-verification|false
if the option is removed from the UI there is no row.
For the export json files, it is the same:

/bin/grep -A 1 -i remote sftp-duplicati-config-opttrue.json 
        "Name": "--full-remote-verification",
        "Value": "true",
~/Téléchargements$ /bin/grep -A 1 -i remote sftp-duplicati-config-optfalse.json
        "Name": "--full-remote-verification",
        "Value": "false",
~/Téléchargements$ /bin/grep -A 1 -i remote sftp-duplicati-config-noconfig.json 

So there should not be any problem when using the Web UI. I think that the problem you are referring to could be with people scripting the command line directly. I have tried to address this case with this update:

                string policy;
                if (!m_options.TryGetValue("full-remote-verification", out policy))
                    policy = "False";

                RemoteTestStrategy r;
                if (!Enum.TryParse(policy, true, out r))
                    r = RemoteTestStrategy.True;

                return r;